catchpoint-mixin/variables.libsonnet

```diff
@@ -70,7 +70,7 @@ local utils = commonlib.utils;
 
       testNameVariable:
         [root.datasources.prometheus]
-        + variablesFromLabels(groupLabels, instanceLabels, filteringSelector, multiInstance=true) + [testNameLabel],
+        + variablesFromLabels(groupLabels, instanceLabels + testNameLabel, filteringSelector, multiInstance=false),
```

Comment:
I think having the Test_name as a selector is good. I think because there are potentially so many node names though that we need to include a Node_name multiInstance selectable too so it is not as cluttered.